lib/Extension/Core/Model/JsonSchemaBuilder.php

```diff
@@ -28,7 +28,7 @@ public function __construct(string $title, array $extensions)
     public function dump(): string
     {
         $schema = [
-          '$schema' => 'https =>//json-schema.org/draft/2020-12/schema',
+          '$schema' => 'https://json-schema.org/draft-07/schema',
```

Comment:
i think the original is valid without the `=>` typo: https://json-schema.org/learn/getting-started-step-by-step

Reply:
Probably it's valid but it does not work for me for some reason, in opposite to `draft-07`. Maybe it's a problem of vscode-json-language-server. It reports that this draft it not fully supported yet. 

It's not important as we do not edit the schema file manually in normal case. If you prefer `2020-12` then no problem for me to change it back.

Reply:
no it's fine i guess